src/js_crypto/test_vectors/dump-test-vectors.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+pushd ../../../../../crypto/proof-systems/oracle/export_test_vectors
+  cargo run -p export_test_vectors -- b10 kimchi ../../../../snarky_js_bindings/snarkyjs/src/js_crypto/test_vectors/testVectors.json
```

Comment:
You could also just generate them offline and save them to the repo somewhere, so that you don't need to have this dependency.

Reply:
Hm I didn't think of that :D Sounds like an idea